Source/Platform/DefaultOpenTK/Backend/Graphics/NativeWindow.cs

```diff
@@ -274,5 +275,30 @@ private void OnRenderFrame(FrameEventArgs e)
 			Profile.TimeSwapBuffers.EndMeasure();
 			Profile.TimeRender.EndMeasure();
 		}
+
+		public void SetCursor(ContentRef<Pixmap> cursor, Point2 hotspot)
+		{
+			if (!cursor.IsAvailable)
+			{
+				Logs.Core.WriteWarning("SetCursor: pixmap {0} not available", cursor.Name);
+			}
+			else if (this.internalWindow != null && this.internalWindow.CursorVisible)
```

Comment:
What about cases where the cursor is set first, and _then_ it is made visible?

Reply:
Removed check, but the way the backend is currently implemented, setting SystemCursorVisible to false changes the Cursor to Empty.. not sure if it's the right approach or if it's better to work on the correct NativeWindow property...